tests/integration/tests.mk

```diff
@@ -48,7 +48,7 @@ test.integration.all: test.integration test.integration.kube
 
 # Generate integration test targets for kubernetes environment.
 test.integration.%.kube:
-	$(GO) test -p 1 ${T} ./tests/integration/$*/... ${_INTEGRATION_TEST_WORKDIR_FLAG} ${_INTEGRATION_TEST_CIMODE_FLAG} -timeout 30m \
+	$(GO) test -p 1 ${T} ./tests/integration/$*/... ${_INTEGRATION_TEST_WORKDIR_FLAG} ${_INTEGRATION_TEST_CIMODE_FLAG} -timeout 10m \
```

Comment:
Why reduce the timeout?

Reply:
What I found from this PR is that all 4(?) of the security tests have the same issue, not just the two I disabled. 30min is very long when the hard cap on prow is 2hrs (not sure if that is something we configured or not, but the tests die at 2hrs). afaik the huge timeout was for some bug with image pulling on prow which has been fixed. If a test is taking more than 10min I think it is pretty safe to consider it an error, but we could bump this back.

I think we should either keep it low, or disable all the security tests (or, ideally, fix the tests).